### DISCUSSION TITLE
Co-leading Rooms

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#151] [Coffees]
@Virtual-Coffee/coffee-talk-team, for those of you who have co-led a room:

- What has gone well or what have you liked?
- What could be added or improved?
- What haven't you liked?

**Comment by BekahHW** [comment]
- I've really liked seeing other people's styles of facilitating, and seeing how the others in the room respond to different facilitation types. I've learned from every person I've co-led with.
- Sometimes it feels a little like stepping on toes, and I don't want it to be that way. I think when we're onboarding new room leaders it will feel less like this. But having a conversation with co-leads beforehand seems to help.

**Comment by SuzeShardlow** [comment]
I think it would be useful to outline the specific problems we're trying to solve with this - the whys.

Then figure out what exactly we need to do to solve the problem. Specific tasks etc.

I think if we did that, it wouldn't feel like stepping on toes.

I haven't co-led yet. I've read the info in the channel and tbh it wasn't clear how it would work in practice with two people hosting, which goes back to your stepping on toes comment.

**Reply by kierblk** [reply]
I like outlining the why/concerns we are aiming to address.

**Reply by cafloyd** [reply]
I wonder if it might even be reasonable to define a few group goals, but then allow each pair to identify their main goal from that list. For example, if there is a new room leader, the pair's main goal might be to onboard the new leader; which might be a slightly different approach than if there are two veteran room leaders who want to focus on splitting up between spoken conversation and the chat in order to engage everyone in the room.

**Comment by SuzeShardlow** [comment]
I like the idea of this being a means of giving and getting feedback and would be interested to know if folks have received any that surprised and/or helped them 🙂

**Comment by jonathanyeong** [comment]
I've only been in one co-lead room, and I thought it was great! The responsibilities are still a little hazy for me. But I like having a co-lead to help facilitate and drive discussion. I sometimes struggle to keep the conversation going and have felt the awkward silences when I've lead a room by myself before.

**Comment by glenmccallumcan** [comment]
First time co-leading this week for me. due to some technical difficulties I started off the room. And then I continued controlling most of the discussion due to momentum. my apologies to @cafloyd . I classify yesterday as not going well from a co-leading standpoint

I think if the purpose of co-leads is for onboarding then the new lead should run everything by default. And the other lead is there as an insurance policy in case things drop off or go sideways. And .... when things go sideways it's really nice having backup (like @mrsantons in my room last week ).

I'd like another week with @cafloyd so I can totally back off and let her run with it.

**Reply by SuzeShardlow** [reply]
Agree - comes back to defining the why 😀

**Reply by BekahHW** [reply]
In my mind, there are a couple of reasons why, but I'd be up for feedback on this as well: 
- onboard new room leaders
- allow existing room leaders to provide each other with feedback
- allow room leaders to learn from each other (I mean this separately from feedback, but by being able to watch, listen, and experience different ways of room leading)
- allow for more support for current room leaders. For example, if I have a hard time knowing how to address someone who dominates the conversation, then I can communicate that to my co-lead beforehand and ask for their support when that happens or ask them for tips.

**Reply by SuzeShardlow** [reply]
Speaking to those whys and also the comments about stepping on toes/confusion about who's doing what, it sounds like it might work better if the co-leader shadows the leader and only steps in if needed.  I think going in with the intention of splitting the actual leading/facilitating adds more cognitive load to an already fast-moving role.

Also, onboarding is different from getting trained in an existing role, so probably needs a different solution which looks at how leaders are identified/recruited/put themselves forward and what they bring to the role vs what they need to learn.  Eg when they are recruited, have they already attended coffees, have they already observed existing leaders?  Or do they have prior skills and experience which means they are OK to jump in?  Either one of those scenarios is great.  But folks who don't fit into either of those personas will need to do the observing part for a while so that they can learn some tools for when they do it for real.

**Reply by kierblk** [reply]
I definitely think there is a difference to on boarding someone new and existing room leads growing more into the role. I feel a more rigid process for on boarding would be helpful while having the more tenured room leads to decided the best way to learn from each other/support each other while leading the room could be best.

**Reply by BekahHW** [reply]
@SuzeShardlow when Tom and I co-lead, this is how we split roles which seems to make it clearer:

Co-Lead 1: Reads the intro script and starts intros. Monitors chat during the conversation.
Co-Lead 2: Initiates the discussion and facilitates the discussion.

But I think this worked pretty well for us, but I'm not sure this would work for everyone. The goal is to support and provide feedback, not to stress people out.

**Reply by cafloyd** [reply]
@glenmccallumcan - no apologies necessary! I enjoyed being in the break-out room with you!

I think our challenges came about because I subbed in for another co-lead at the last minute, and then joined the room late due to technical difficulties. Since we planned for me to do the opening script and icebreaker, it was hard for us to recalibrate when I joined at the end of the icebreaker. The discussion was also flowing really naturally, and there wasn't a lot of need for us to jump in too much. I think if we had defined our goal as providing support if discussion got overwhelming, it would have been fine that I was hanging back, since the pace of conversation was good and everyone was participating. If we had defined our goal as onboarding me, we might have split up our roles a little differently. Ultimately though, while the execution was a little awkward, I think everyone in our room got a lot out of the discussion!

In general though, I agree with everyone that's said that having a clear sense of the goal with having co-leads would make it easier to adjust to these kinds of unforeseen hurdles! I think it's a great idea to have co-leads, both for onboarding purposes and to get a chance to see other room leader styles, like @BekahHW mentioned!

**Comment by BekahHW** [comment]
@saramccombs @tkshill @tomcudd I'd love to have your thoughts here if you have time.

**Comment by bogdaaamn** [comment]
I like the means of giving feedback on the leading practices. This was something that I missed before, I never knew how I am doing and if I should change something, and never had the courage to ask (the notetaker for example). 

I co-lead with @saramccombs last week and I really enjoyed it. I like it takes some pressure off moderating the conversation honestly.

**Comment by tkshill** [comment]
I agree with Glen. Two people leading in tandem can take a little bit of rhythm so I like lead/colead as one person clearly doing the conversation facilitation, and maybe the other doing chat moderation and providing backup support. The part I enjoy the most is simply having another person I know is there to support. With the notetaker also being that stabilizer, but it's hard to code switch when notetaking.

We can adjust the documentation to make it clear the distinctions between the roles. With the caveat that the leads can always adjust to what's comfortable for the both of them. I remember one time Seth had some connection troubles, so I pivoted into the lead role for a bit while he got that sorted. We were just room lead and note taker but that type of thing is what I think we're trying to accomplish here. No one person feels like they have to be perfect.

Also, having a person to have feedback with is great, but not everyone can touch base immediately after zoom sessions. So lead pairs would have to self organize some time to do that. It could just be through a thread in the room-leader/note-taker channels.

**Reply by BekahHW** [reply]
I've mostly been doing it through DMs, bc that's been easiest.

It might be nice to have some support for how to provide feedback. Sometimes that can be a real challenge, and I'd like to create a way for everyone to give feedback without being afraid of hurting people's feelings.

**Reply by cafloyd** [reply]
In previous jobs, I've had to provide feedback to teachers, which is a different but similar situation. It can be really tricky at first, and having some kind of guideline for how to do it can be really helpful!

**Reply by nickytonline** [reply]
Same as Kirk and Tom said. I've only done it once so far, but it felt like stepping on toes unintentionally. I'm also not certain we need to folks to lead a breakout room.

**Reply by MikeRogers0** [reply]
>  I'm also not certain we need to folks to lead a breakout room.

I kind of see the lead as a the person who kick starts the breakout room & starts the conversation fly wheel, while also keeping an eye on people talking to much.

**Comment by tomcudd** [comment]
I've thought about this a bit. I enjoy the fact that we are stretching and pushing the edges of our comfort to ensure that we all can be our best selves. Honestly, it serves two really great purposes: making sure we have both support and accountability for the people who are leading rooms, and the preparing us for growth of new leads. I'm not necessarily ready to completely define every specific part of what the co lead looks like, as it may organically arise after a few more sessions and we see what fits.

**Comment by meg-gutshall** [comment]
Hey all! I finally got a chance to read through this discussion thread! So far I have been a notetaker in a room that had co-leads and was a co-leader of another room. 

## My Own Experience

From my personal experience, it appears that for both parties to benefit from co-leading, it will take checking in with each other both before and after the actual coffee meeting. I did neither of these things when co-leading nor did my co-lead, so it went off like a normal coffee where I led and they kind of hung back. I think that's partly because I'm becoming increasingly more crunched for time and also there wasn't a clear definition of what "checking in" meant. The conversation above gives me a better idea now.

## My Takeaways from the Above Thread

A few things discussed here stand out to me. I agree with @SuzeShardlow in that it's important for us to "define the why" behind co-leading breakout rooms and I believe @BekahHW shared [several good points](https://github.com/Virtual-Coffee/VC-Contributors/discussions/151#discussioncomment-798391) which go into said definition. The first two bullet points seem like the primary reasons for using co-leads (to me, the second bullet point seems to encompass the final bullet point).

Also @cafloyd, I'd be interested to hear more about your experience with providing feedback and if there was something your employer did to encourage honesty without hurt feelings. For that matter, any manager here or person who has done code reviews would probably have transferrable insights as to how to best provide constructive feedback.

Personally, I like the way @tomcudd and @BekahHW decided to split up the co-lead roles. I recently attended a study group in which one instructor was live coding while the other was dropping links and answering questions in the chat. You could still ask questions out loud, but if it was a quick one or maybe topic-adjacent or you just didn't want to ask out loud, you could do it in chat! This structure flowed nicely and seemed to work for them as well!

As to @tkshill's point, I think (especially for those still getting used to leading a room) knowing that there's support in case shit hits the fan can give a significant confidence boost and make the meeting go smoother overall.

## Another Idea to Throw Into the Mix

When we had our meeting a while back and discussed notetakers sharing their notes link with the room leader, this could be an opportunity to add to that process as well. As a room leader, it's difficult to remember to take notes while the notetaker is speaking and as a notetaker you have to remember what you said and add it in later or else trash it. With co-leads that could make things a bit easier on everyone in this sense!
